ui/de-lib/src/main/java/org/iplantc/de/client/models/groups/Member.java

```diff
@@ -0,0 +1,52 @@
+package org.iplantc.de.client.models.groups;
+
+import com.google.gwt.user.client.ui.HasName;
+import com.google.web.bindery.autobean.shared.AutoBean;
+import org.iplantc.de.client.models.HasId;
+
+import java.util.List;
+
+/**
+ * @author dennis
+ */
+public interface Member extends HasId, HasName {
+
+    @AutoBean.PropertyName("id")
+    void setId(String id);
+
+    @AutoBean.PropertyName("first_name")
+    String getFirstName();
+
+    @AutoBean.PropertyName("first_name")
+    void setFirstName(String firstName);
+
+    @AutoBean.PropertyName("last_name")
+    String getLastName();
+
+    @AutoBean.PropertyName("last_name")
+    void setLastName(String lastName);
+
+    @AutoBean.PropertyName("email")
+    String getEmail();
+
+    @AutoBean.PropertyName("email")
+    void setEmail(String email);
+
+    @AutoBean.PropertyName("institution")
+    String getInstitution();
+
+    @AutoBean.PropertyName("institution")
```

Comment:
By the way, these PropertyNames for `id`, `email`, and `institution` are redundant.
The PropertyName only needs to be specified if the JSON key doesn't match (case insensitive) the method name after the get/set. So the PropertyNames with the underscores in them are required.